Sources/AppcuesKit/Presentation/Traits/TraitComposer.swift

```diff
@@ -18,12 +18,14 @@ internal class TraitComposer: TraitComposing {
 
     private let traitRegistry: TraitRegistry
     private let actionRegistry: ActionRegistry
+    private let config: Appcues.Config
     private let notificationCenter: NotificationCenter
 
     init(container: DIContainer) {
         traitRegistry = container.resolve(TraitRegistry.self)
         actionRegistry = container.resolve(ActionRegistry.self)
         notificationCenter = container.resolve(NotificationCenter.self)
+        config = container.resolve(Appcues.Config.self)
```

Comment:
I'll add the dependency to the whimsical graph

Reply:
thank you!